### PR TITLE
rpmqa: ignore gpg-pubkey, not gpg-pubkey-

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: atomic_reactor/plugins/post_rpmqa.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def run(self):
         # gpg-pubkey- are autogenerated packages by rpm when you import a gpg key
         # these are of course not signed, let's ignore those by default
         if self.ignore_autogenerated_gpg_keys:
-            self.log.debug("ignore rpms 'gpg-pubkey-'")
-            plugin_output = [x for x in plugin_output if not x.startswith("gpg-pubkey-")]
+            self.log.debug("ignore rpms 'gpg-pubkey'")
+            plugin_output = [x for x in plugin_output if not x.startswith("gpg-pubkey")]
```
Comment (twaugh) [inline atomic_reactor/plugins/post_rpmqa.py]: Should it be looking for the delimiting comma as well though?

File: atomic_reactor/plugins/post_rpmqa.py [hunk 2]
```diff
@@ -40,8 +40,8 @@ def run(self):
         # gpg-pubkey- are autogenerated packages by rpm when you import a gpg key
```
Comment (twaugh) [inline atomic_reactor/plugins/post_rpmqa.py]: How about updating this comment?

File: atomic_reactor/plugins/post_rpmqa.py (hunk 1 above)
Comment (TomasTomecek) [inline atomic_reactor/plugins/post_rpmqa.py]: hmmm, good point

File: atomic_reactor/plugins/post_rpmqa.py (hunk 2 above)
Comment (TomasTomecek) [inline atomic_reactor/plugins/post_rpmqa.py]: another /facepalm
will address this later, thanks Tim!

**Pull request merged**